vformer/models/classification/convvt.py

```diff
@@ -0,0 +1,127 @@
+import torch
+import torch.nn as nn
+from timm.models.layers import trunc_normal_
+
+from ...encoder.convvt import ConvVTStage
+from ...utils import MODEL_REGISTRY
+
+
+@MODEL_REGISTRY.register()
+class ConvVT(nn.Module):
```

Comment:
How about we keep the name the authors use, `CvT`? Would help in the model implementation being easily identifiable. Will need to change the file names, and names in the test files as well.

Reply:
I used ConvVT because there's already a [CVT](https://github.com/SforAiDl/vformer/blob/main/vformer/models/classification/cvt.py) (compact instead of conv)